### ISSUE TITLE
Dashboard Updates - Ministries with Sub-Orgs

### ISSUE THREAD

**Issue by m-prodan** [bcgov/foi-flow#2212]
* As a dashboard user
* I want to see the overall, and breakdown of a Ministry with sub-orgs
* so that I can see how my overall Ministry is performing, and its sub-orgs

**Assumptions & Scope**	
This story reflects the need for users of our dashboards to see a breakdown

What is IN scope?
Updating visualizations of the dashboards for Ministries that contain sub-orgs.
In scope for this activity includes:
All Dashboards that include the ability to filter by Ministry/Processing Org
Ministry of Finance and its sub-orgs (CAS, LDB, GCP, PSA)
Ministry of Environment and its sub-org (EAO)
Ministry of Public Safety and Solicitor General and its sub-orgs (EMB, OCC)
Ministry of Indigenous Relations and Reconciliation and its sub-orgs (DAS)

What is NOT in scope?
Other feedback received about the dashboard - will be covered in seperate user stories. 

**Acceptance Criteria**	

**Scenario 1: Overall View (all Ministries) - Performance Dashboard**
* GIVEN a user has accessed the Performance Dashboard
* WHEN the graphs populate
* THEN the overall performance metrics for Finance, Environment, Public Safety and Indigenous Relations will be an overall metric that includes all of their respective sub-orgs

**Scenario 2: Received Dashboard**
* GIVEN a user has accessed the Received Dashboard
* WHEN the graphs populate
* THEN the overall received metrics for Finance, Environment, Public Safety, and Indigenous Relations will be an overall metric that includes all of their respective sub-orgs

**Scenario 3: Ministry Filter with Sub-Orgs - Performance Dashboard**
* GIVEN a user has accessed the performance dashboard
* WHEN they filter to a Ministry with sub-orgs (eg. Finance)
* THEN the metrics that display will include each sub-org, and an overall metric for the Ministry

**Scenario 4: Ministry Filter with Sub-Orgs - Received Dashboard**
* GIVEN a user has accessed the received dashboard
* WHEN they filter to a Ministry with sub-orgs (eg. Finance)
* THEN the metrics that display will include each sub-org, and an overall metric for the Ministry

**Dependencies? What is the impact of this dependency? (If so, link dependency in the ticket, make it visible in a team´s backlog)**

**Validation Rules? (If yes, list here)**	

**Design**
@xxx - please link the Design here

**Definition of Ready**

1. [ ] Is there a well articulated User Story?
2. [ ] Is there Acceptance Criteria that covers all scenarios (happy/sad paths)?
3. [ ] If there is a user interface, is there a design? 
4. [ ] Does the user story need user research/validation?
5. [ ] Does this User Story needs stakeholder approval?
6. [ ] Design / Solution accepted by Product Owner
7. [ ] Is this user story small enough to be completed in a Sprint? Should it be split?
8. [ ] Are the dependencies known/ understood? (technical, business, regulatory/policy)
9. [ ] Has the story been estimated?

**Definition of Done**

1.    [ ] Passes developer unit tests
2.    [ ] Passes peer code review
3.    [ ] If there's a user interface, passes UX assurance
4.    [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
5.	[ ] Confirm Test cases built and succeeding 
6.	[ ] No regression test failures
7.	[ ] Test coverage acceptable by Product Owner
8.	[ ] Ticket ready to be merged to master or story branch
9.	[ ] Developer to list Config changes/ Update documents and designs
10.	[ ] Can be demoed in Sprint Review 
11.	[ ] Tagged as part of a Release
12.   [ ] Feature flagged if required
13.  [ ] Change Management activities done?

**Comment by abin-aot** [comment]
Re-estimated, due to more effort.

**Comment by divyav-aot** [comment]
@m-prodan do you want the changes only for performance and received dashboards? or you want the similar changes to Analyst Workload Dashboard and Active OIPC Review Report. I can see ECON has these dashboards as well?

**Comment by m-prodan** [comment]
@divyav-aot - Active OIPC review reports is fine as is - but I do recall one piece of feedback for that is it should NOT show OIPC reviews in the closed state. I am unsure if that was addressed but we can fix in a future ticket if not.

Analyst workload dashboard is also fine as is, as Ministry will select by processing org anyways. I did just notice in staging that the analyst workload dashboard appears to only be showing closed requests. This isn't correct, it should just be open requests. I don't have a filter to adjust this unfortunately, so we may need to create a new ticket to fix it
